skosprovider/skos.py

```diff
@@ -531,6 +531,7 @@ def label(labels=[], language='any', sortLabel=False):
         return None
     if isinstance(language, str):
         language = [language]
+    language = [l for l in language if tags.tag(l).language] if language else language
```

Comment:
It might be more appropriate to raise an exception for unexisting languages instead of treating it as 'undefined'

Reply:
I discussed with @goessebr and unexisting language will default to 'undefined' because the exception would consist of a larger refacter. Probably with an issue in pyramid_skosprivder, because there is the language query param processed